packages/react-admin-color-picker/src/core/ColorPicker.styles.ts

```diff
@@ -0,0 +1,112 @@
+import { Theme } from "@material-ui/core";
+import { createStyles } from "@material-ui/styles";
+
+export type VPAdminColorPickerClassKeys =
+    | "wrapper"
+    | "field"
+    | "pickedColorIndicator"
+    | "pickerWrapper"
+    | "saturationWrapper"
+    | "saturationPointer"
+    | "hueWrapper"
+    | "hueSliderMarker"
+    | "paletteWrapper"
+    | "colorTile";
+
+const styles = (theme: Theme) =>
+    createStyles({
+        wrapper: {
+            position: "relative",
+            width: "100%",
+            height: "100%",
+        },
+        field: {
+            position: "relative",
+            height: "32px",
+            display: "flex",
+            flexDirection: "row",
+            justifyContent: "flex-start",
+            alignItems: "center",
+            padding: "0 10px",
+            border: `1px solid ${theme.palette.grey[300]}`,
+            borderRadius: "2px",
+            cursor: "pointer",
+        },
+        pickedColorIndicator: {
+            width: "20px",
+            height: "20px",
+            marginRight: "10px",
+            borderRadius: "5px",
```

Comment:
Da machts glaub ich Sinn, den allgemeinen Wert vom Theme zu verwenden:
```suggestion
            borderRadius: ${theme.shape.borderRadius},
```